docs/source/getting-started.rst

```diff
@@ -73,7 +73,8 @@ metrics.
 This is what a metric could look like:
 
 .. figure:: _static/screenshot.png
-    :alt: Metric example
+    :target: _static/screenshot.png
```

Comment:
Note I didn’t test this.

Reply:
This seems to work: https://django-postgres-metrics--58.org.readthedocs.build/en/58/getting-started.html